tests/functional/spec/census-communal.spec.js

```diff
@@ -54,5 +54,27 @@ describe('Example Test', function() {
       expect(ThankYou.isOpen()).to.be.true
   })
 
+  it('Given I am completing the Communal Establishment questionnaire, When I answer No to the Usual Residents question, Then I should be routed to Completion Preferences', function() {
+
+      // Given I am completing the Communal Establishment questionnaire,
+      openQuestionnaire('census_communal.json')
+
+      EstablishmentType.clickEstablishmentTypeAnswerHotel().submit()
+      BedSpaces.setBedSpacesAnswer(2).submit()
+
+      // When I answer No to the Usual Residents question
+      UsualResidents.clickUsualResidentsAnswerNo().submit()
+
+      // Then I should be routed to Completion Preferences
+      CompletionPreferenceEstablishment.clickCompletionPreferenceEstablishmentAnswerPaper().submit()
```

Comment:
@iwootten raised a good point yesterday, in that these tests don't need to reach the end of the survey. you only need to test that you get to the CompletionPreferenceEstablishment page. this would save us lots of build time.